proto/data_test.go

```diff
@@ -219,11 +219,11 @@ func TestPrevKey(t *testing.T) {
 }
 
 func TestKeyString(t *testing.T) {
-	if KeyMax.String() != "\xff..." {
+	if KeyMax.String() != "\\xff..." {
```

Comment:
Nice. I had looked for an open-sourced version of OrderedCode before but couldn't find it. I would like to move to this scheme because this String() method doesn't catch all of the places where we try to log 4000 repetitions of `\xff` (this happens to me a lot whenever I turn on verbose raft logging).